.gitignore

```diff
@@ -1,4 +1,5 @@
 node_modules
 .nyc_output
+.vscode
```

Comment:
Adding this line to `.gitignore` is just because VSCode automatically created this folder when I opened this repo.